frontend/src/views/inventory/AddProductView.vue

```diff
@@ -28,6 +28,7 @@
             const productModalShow = ref(false)
             const productModalShowError = ref(false)
             const msm400 = ref('')
+            const checked = ref(false)
             const router = useRouter()
             const formStatus = ref<Status>({
                 id: 0,
```

Comment:
Consider the use of ```composition API``` instead of ```options API```.
see:
[Typescript with Composition API](https://vuejs.org/guide/typescript/composition-api.html)